src/main/java/range/RangeCreater.java

```diff
@@ -0,0 +1,45 @@
+package range;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RangeCreater {
+
+	public static void getRangeReading(String values) {
```

Comment:
Where is this method used?
Please let me know if there is a test case covering this method...